config/config.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import (
+	"fmt"
+
+	"gorm.io/gorm"
+)
+
+var db *gorm.DB
+
+func Init() error {
+	var err error
+
+	db, err = initializeDatabase()
+
+	if err != nil {
+		return fmt.Errorf("error initializing database: %v", err)
```

Comment:
on main.go you used `log` so we should change all `fmt` to use `log`, what do you think?

Reply:
I use `fmt` for wrapping up the error and bubbling back to the caller , so when we hit the `log.Fatal`, all the errors are printed on a cascade, like so:

![image](https://github.com/forjadev/gun-organization/assets/41487169/6466cb96-79c2-46e5-9417-b5c63b5ca278)

This way, we can trace where the problem is coming from, like a stack trace. But this can be bad as our project increases, since the terminal would be flooded. I think both `log` and `fmt` options are good. Wby?

Reply:
Makes sense, no problem to me!

Reply:
I think we can create a custom log later and replace everywhere when our project get bigger